### PR TITLE
Introducing minishift - run openshift locally

### PR DESCRIPTION
**Pull request by budhrg**
Hi @jberkus ,

Please review the changes for the blog. I have updated the blog post with all the required changes.

Please let us know if there is any changes still required.

### CODE REVIEW

**Comment by praveenkumar** [comment]
LGTM 👍

**Comment by jberkus** [comment]
OK, I'm at Summit right now.  Can someone else do a complete run-through of the technical instructions, to make sure there are no typos etc.?  I probably won't be able to.

File: source/blog/2017-05-03-minishift-intro.html.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+## Installing Minishift
+
+1. Install your preferred VM platform on your workstation.
+
+    **Note**: You need to install additional docker-machine driver for 'kvm' or 'xhyve'. See [Docker Machine drivers](https://docs.openshift.org/latest/minishift/getting-started/docker-machine-drivers.html) guide to install it properly.
+
+1. Download the archive for your OS from the Minishift [releases page](https://github.com/minishift/minishift/releases/tag/v1.0.0) and unpack it.
+1. Copy the contents of the directory to your preferred location.
+1. Add the minishift binary to your _PATH_ environment variable.
```
Comment (jlebon) [inline source/blog/2017-05-03-minishift-intro.html.md]: Numbering seems off here.

File: source/blog/2017-05-03-minishift-intro.html.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+    $ minishift oc-env
+    export PATH="/Users/john/.minishift/cache/oc/v1.5.0:$PATH"
+    # Run this command to configure your shell:
+    # eval $(minishift oc-env)
+    ```
+
+1. Run `eval $(minishift oc-env)` as per above instructions to setup your shell.
+    ```
+    $ eval $(minishift oc-env)    # Might differ based on OS and shell type
+    ```
```
Comment (jlebon) [inline source/blog/2017-05-03-minishift-intro.html.md]: Numbering is off in this list too.

This step looks like a repeat of the previous step, no?

Comment (budhrg) [inline source/blog/2017-05-03-minishift-intro.html.md]: Same here as well. 
screenshot from 2017-05-03 19-56-15

Markdown take care of numbering when we list the items with 1. ...  1.... 1...  I believe.

File: source/blog/2017-05-03-minishift-intro.html.md (hunk 1 above)
Comment (budhrg) [inline source/blog/2017-05-03-minishift-intro.html.md]: @jlebon Rendering well for me. 
screenshot from 2017-05-03 19-55-15

File: source/blog/2017-05-03-minishift-intro.html.md (hunk 2 above)
Comment (jberkus) [inline source/blog/2017-05-03-minishift-intro.html.md]: @jlebon you have to look at the rendered version.  Markdown autmatically renumbers thing.

**Pull request merged**